app/models/user.rb

```diff
@@ -3,8 +3,12 @@ class User < ApplicationRecord
   has_many :videos, through: :user_videos
 
   validates :email, uniqueness: true, presence: true
-  validates :password, presence: true
+  validates :password, presence: true, on: :update, allow_blank: true
```

Comment:
kept having issues with the user unable to be updated.

Reply:
okie dokie